### ISSUE TITLE
Line spacing question

### ISSUE THREAD

**Issue by mck1777** [ExLibrisGroup/spineomatic-cloudapp#14]
I now have the volumes printing, but was asked if I could remove the line spacing between the call number and the volume indicator. Can you direct me on how to accomplish this, I am not seeing how to change it in the template area.  I have uploaded a pdf of the print job.
[MCRL with v.pdf](https://github.com/ExLibrisGroup/spineomatic-cloudapp/files/6297585/MCRL.with.v.pdf)

Thanks
maryclare koenig

**Comment by jweisman** [comment]
Thanks for your question. Can you provide some example barcodes and the environment you're using so we can check it?

**Comment by mck1777** [comment]
Not sure what you mean by environment, I am working in Chrome Version 89.0.4389.114 (Official Build) (64-bit) 
My laptop is using Windows 10 Enterprise version 1909

We are having issues with Firefox not clearing margins, but that is a network setting I am trying to get IET to resolve. I am attaching an excel file with MMS ID and barcodes along with the pdf of the printed labels

What elese can I provide to assist you

[example 2b.pdf](https://github.com/ExLibrisGroup/spineomatic-cloudapp/files/6350768/example.2b.pdf)
[example 2b.xlsx](https://github.com/ExLibrisGroup/spineomatic-cloudapp/files/6350770/example.2b.xlsx)

**Comment by jweisman** [comment]
Sorry for the confusion- I meant which Alma environment can we check in? Sandbox? Institution? Thanks!

**Comment by mck1777** [comment]
:)
Institution.
mck

**Comment by jweisman** [comment]
Thanks. I was able to take a look. In order to create a line break without any space, hold down "Shift" and press "Return/Enter". You see there is now no space in the template.

![Screen Shot 2021-04-22 at 15 35 23 ](https://user-images.githubusercontent.com/80420/115715151-8d7f3e80-a380-11eb-8d50-8ff81b11b349.png)

I updated the Mcrl V 1 template, saved it, and printed out some barcodes. I think this is what you're looking for.

![Screen Shot 2021-04-22 at 15 34 30 ](https://user-images.githubusercontent.com/80420/115715179-9243f280-a380-11eb-9aed-f62a94850f05.png)

Hope this helps.
